api/src/main/java/uk/gov/dhsc/htbhf/hmrc/service/EligibilityService.java

```diff
@@ -32,7 +32,7 @@ public EligibilityService(HouseholdRepository repository,
     public EligibilityResponse checkEligibility(HMRCEligibilityRequest eligibilityRequest) {
         Optional<Household> household = repository.findHouseholdByAdultWithNino(eligibilityRequest.getPerson().getNino());
         if (household.isPresent()) {
-            log.debug("Matched CTC household: {}", household.get().getHouseholdIdentifier());
+            log.debug("Matched CTC household: {}", household.get().getId());
```

Comment:
Same here, are we logging DEBUG to Production logs? If not then we don't need this change.